ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+};
+
+WDAPI.Driver.prototype.back = function() {
+  return this.ref + ".navigate().back()";
+};
+
+WDAPI.Driver.prototype.close = function() {
+  return this.ref + ".close()";
+};
+
+WDAPI.Driver.prototype.findElement = function(locatorType, locator) {
+  return new WDAPI.Element(this.ref + ".findElement(" + WDAPI.Driver.searchContext(locatorType, locator) + ")");
+};
+
+WDAPI.Driver.prototype.findElements = function(locatorType, locator) {
+  return new WDAPI.ElementList(this.ref + ".findElements(" + WDAPI.Driver.searchContext(locatorType, locator) + ")");
+};
+
+WDAPI.Driver.prototype.getCurrentUrl = function() {
+  return this.ref + ".getCurrentUrl()";
+};
+
+WDAPI.Driver.prototype.get = function(url) {
+  if (url.length > 1 && (url.substring(1,8) == "http://" || url.substring(1,9) == "https://")) { // url is quoted
+    return this.ref + ".get(" + url + ")";
+  } else {
+    return this.ref + ".get(baseUrl + " + url + ")";
+  }
+};
+
+WDAPI.Driver.prototype.getTitle = function() {
+  return this.ref + ".getTitle()";
+};
+
+WDAPI.Driver.prototype.getAlert = function() {
+  return "closeAlertAndGetItsText()";
+};
+
+WDAPI.Driver.prototype.chooseOkOnNextConfirmation = function() {
+  return "acceptNextAlert = true";
+};
+
+WDAPI.Driver.prototype.chooseCancelOnNextConfirmation = function() {
+  return "acceptNextAlert = false";
+};
+
+WDAPI.Driver.prototype.refresh = function() {
+  return this.ref + ".navigate().refresh()";
+};
+
+WDAPI.Element = function(ref) {
+  this.ref = ref;
+};
+
+WDAPI.Element.prototype.clear = function() {
+  return this.ref + ".clear()";
+};
+
+WDAPI.Element.prototype.click = function() {
+  return this.ref + ".click()";
+};
+
+WDAPI.Element.prototype.getAttribute = function(attributeName) {
+  return this.ref + ".getAttribute(" + xlateArgument(attributeName) + ")";
+};
+
+WDAPI.Element.prototype.getText = function() {
+  return this.ref + ".getText()";
+};
+
+WDAPI.Element.prototype.isDisplayed = function() {
+  return this.ref + ".isDisplayed()";
+};
+
+WDAPI.Element.prototype.isSelected = function() {
+  return this.ref + ".isSelected()";
+};
+
+WDAPI.Element.prototype.sendKeys = function(text) {
+  return this.ref + ".sendKeys(" + xlateArgument(text) + ")";
+};
+
+WDAPI.Element.prototype.submit = function() {
+  return this.ref + ".submit()";
+};
+
+WDAPI.Element.prototype.select = function(selectLocator) {
+  if (selectLocator.type == 'index') {
+    return "new Select(" + this.ref + ").selectByIndex(" + selectLocator.string + ")";
+  }
+  if (selectLocator.type == 'value') {
+    return "new Select(" + this.ref + ").selectByValue(" + xlateArgument(selectLocator.string) + ")";
+  }
+  return "new Select(" + this.ref + ").selectByVisibleText(" + xlateArgument(selectLocator.string) + ")";
+};
+
+WDAPI.ElementList = function(ref) {
+  this.ref = ref;
+};
+
+WDAPI.ElementList.prototype.getItem = function(index) {
+  return this.ref + "[" + index + "]";
+};
+
+WDAPI.ElementList.prototype.getSize = function() {
```

Comment:
Fixed in [#7e0de8f](https://github.com/rasika90/selenium/commit/7e0de8f7253c45ace8d0fd539f4712e1f91349c1).